.travis.yml

```diff
@@ -1,33 +1,52 @@
 language: python
 sudo: false
 env:
-  - PYTHON_VERSION=2.7
+  - PYTHON_VERSION="3.4"
+  - PYTHON_VERSION="3.5"
 before_install:
   - wget http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
   - chmod +x miniconda.sh
   - bash miniconda.sh -b -p $HOME/miniconda
   - export PATH="$HOME/miniconda/bin:$PATH"
   - conda config --set always_yes yes --set changeps1 no
   - conda config --add channels https://conda.anaconda.org/biocore
+  - conda config --add channels https://conda.anaconda.org/mortonjt
```

Comment:
Yes, eventually. First the [RFC](https://github.com/biocore/scikit-bio-rfcs/pull/9) needs to go through ..